src/common/DatePicker/DatePickerView.js

```diff
@@ -1,146 +1,213 @@
-          {calendar.map(({ week }, index) => (
-            <div
-              key={week[0].day.getMonth() + index}
-              className="date-picker_body_week"
-            >
-              {week.map(({ day }) => (
+          <MaskedInput
+            className={inputClassNames}
+            keepCharPositions={true}
+            mask={dateMask}
+            showMask={true}
+            onChange={onInputChange}
+            pipe={autoCorrectedDatePipe}
+            value={valueDatePickerInput}
+          />
+          <span className={inputLabelClassNames}>{label}</span>
+        </div>
+        {openDatePicker && (
+          <div
+            ref={ref}
+            className={`date-picker ${
+              isTopPosition ? 'positionTop' : 'positionBottom'
+            }`}
+          >
+            <div className="date-picker__calendars">
+              {config.map(item => (
                 <div
-                  key={day.getMonth() + day.getDate() + day.getFullYear()}
-                  className={`date-picker_body_week_day ${
-                    date.getMonth() === day.getMonth()
-                      ? 'current-month'
-                      : 'not-current-month'
-                  } ${selectedDate.getDate() === day.getDate() &&
-                    selectedDate.getMonth() === day.getMonth() &&
-                    'selected'}`}
-                  onClick={() => {
-                    date.getMonth() === day.getMonth() && setSelectedDate(day)
-                  }}
+                  className={classnames('date-picker__calendar')}
+                  key={item.id}
                 >
-                  {day.getDate()}
+                  <div className="date-picker__header">
+                    <Arrow
+                      data-testid="btn-previous-month"
+                      className="date-picker__header-previous-month"
+                      onClick={() => onPreviousMonth(item.id)}
+                    />
+                    {isRange && (
+                      <span className="date-picker__header-label">
+                        {item.label}
+                      </span>
+                    )}
+                    <div>
+                      <span className="date-picker__header-month">
+                        {months[item.visibleDate.getMonth()]}
+                      </span>
+                      <span className="date-picker__header-year">
+                        {item.visibleDate.getFullYear()}
+                      </span>
+                    </div>
+                    <Arrow
+                      data-testid="btn-next-month"
+                      className="date-picker__header-next-month"
+                      onClick={() => onNextMonth(item.id)}
+                    />
+                  </div>
+                  <div className="date-picker__weeks">
+                    {weekDay.map((day, index) => {
+                      return (
+                        <div
+                          key={day + index}
```

Comment:
For string concatenation, please prefer using template literals rather than `+` operator.
This way the `+` binary operator is less ambiguous, it is used only for addition of number values. (There is also the _unary_ `+` operator which could attempt to coerce a value to a number (like `+'123'`, `+value`), but then I'd rather use the more explicit `Number(value)`.)
```suggestion
                          key={`${day}${index}`}
```